pyracf/common/logger.py

```diff
@@ -161,7 +161,7 @@ def redact_request_xml(
 
     def redact_result_xml(
         self,
-        xml_string: str,
+        xml_response: Union[str, List[int]],
```

Comment:
My be better to call this `response` instead of `xml_response` since it may not be XML.

Reply:
sure, changed.